grudge/discretization.py

```diff
@@ -79,10 +84,10 @@ class DiscretizationCollection:
     """
 
     def __init__(self, array_context: ArrayContext, mesh: Mesh,
```

Comment:
It might be worthwhile removing the constructor from the publicly documented interface.

Reply:
Done!